solution/ui/regulations/stories/SupplementalContent.stories.js

```diff
@@ -13,7 +15,27 @@ const Template = (args, { argTypes }) => ({
 
 export const Basic = Template.bind({});
 Basic.args = {
-    "title": "43",
+    "api_url": "http://localhost:8000/v2/",
+    "title": "42",
     "part": "433",
-    "sections": ["100", "200", "300"]
+    "sections": ["100", "200", "300"],
+    "getSupplementalContentLegacy": () => Promise.resolve(supplementalContentResponse)
 };
+
+export const EmptyCategories = Template.bind({});
+EmptyCategories.args = {
+    "api_url": "http://localhost:8000/v2/",
```

Comment:
would be nice to eventually mock the response to API calls in storybook so we could test/develop against a storybook story without spinning up the app.  Mock Service Worker might be a good way to do it: https://www.vuemastery.com/blog/mock-service-worker-api-mocking-for-vuejs-development-testing/